packages/cli/src/commands/composite/models.ts

```diff
@@ -0,0 +1,83 @@
+import { Command, type CommandFlags } from '../../command.js'
+import { Flags } from '@oclif/core'
+import Table from 'cli-table3'
+import { readEncodedComposite } from '@glazed/devtools-node'
+
+type CompositeModelInfo = {
+  id: string
+  name: string
+  description: string
+  alias?: string
+}
+
+export default class CompositeModels extends Command<CommandFlags, { compositePath: string }> {
+  static description = 'display the list of models included in a composite'
+
+  static args = [
+    {
+      name: 'compositePath',
+      required: true,
+      description: 'A path to encoded composite definition',
+    },
+  ]
+
+  static flags = {
+    ...Command.flags,
+    'id-only': Flags.boolean({
+      description: `display only model streamIDs`,
+      exclusive: ['table'],
+    }),
+    table: Flags.boolean({
+      description: 'display results in a table',
+      exclusive: ['id-only'],
+    }),
+  }
+
+  async run(): Promise<void> {
+    try {
+      const composite = await readEncodedComposite(this.ceramic, this.args.compositePath)
+      if (this.flags['id-only'] === true) {
+        this.spinner.succeed(JSON.stringify(Object.keys(composite.toParams().definition.models)))
```

Comment:
No spinner display here please
```suggestion
        this.log(JSON.stringify(Object.keys(composite.toParams().definition.models)))
```

Reply:
@pawartur please look at previous comments, it's the third time I'm flagging this.